dashboard/test/ui/README.md

```diff
@@ -85,3 +85,8 @@ Run the eyes tests
 
 - If you're new to [Cucumber](http://cukes.info/), read about [Cucumber scenarios](https://github.com/cucumber/cucumber/wiki/Feature-Introduction), especially the keywords [Given When Then](https://github.com/cucumber/cucumber/wiki/Given-When-Then).
 - When debugging test scripts, it can be helpful to add pauses, such as: `And I wait for 5 seconds`.
+
+## See Also
+
+* [Adding an Eyes Test](../../../docs/testing-with-applitools-eyes.md)
+* [General Testing Instructions](../../../TESTING.md)
```

Comment:
Relative links preserve your current branch as you navigate in the GitHub UI.  Absolute links have the branchname embedded in the URL.  It could be confusing to change versions of the docs in the middle of reading them.